doc/examples/registration/plot_lddmm.py

```diff
@@ -0,0 +1,336 @@
+import numpy as np
```

Comment:
Before starting a proper review, could you please add a docstring at the beginning of the gallery example (take a look at the other examples to see how it's supposed to look like). The doc build is currently failing because of the lack of docstring, and it would help if I were able to look at the generated html page in the checks (I can't at the moment since the build fails).

Reply:
@emmanuelle Thanks for the tip, I've now added that docstring.

That said, it still seems to be failing tests in data/tests/test_data.py and and so it still isn't building. I'm honestly not sure how to fix this. These tests pass when I run them locally. (I guessed it might be because I didn't have tests for the two data files(' corresponding getter functions) so I added those and they also fail in the integration checks, but not when I run them locally.

If you have any thoughts on this I'd love to hear them. I'm at a bit of a loss since these tests run correctly for me.